modules/peridynamics/doc/content/source/kernels/HorizonStabilizedFormIFiniteStrainMechanicsNOSPD.md

```diff
@@ -1,8 +1,8 @@
-# Horizon-stabilized Non-ordinary State-based Peridynamic Finite Strain Mechanics Kernel
+# Form I of Horizon-stabilized Peridynamic Correspondence Finite Strain Mechanics Kernel
 
 ## Description
 
-The `HorizonStabilizedFiniteStrainMechanicsNOSPD` Kernel calculates the residual and Jacobian of the force density integral for self-stabilized non-ordinary state-based peridynamic models under finite strain assumptions.
+The `HorizonStabilizedFormIFiniteStrainMechanicsNOSPD` Kernel calculates the residual and Jacobian of the force density integral for Form I of the horizon-stabilized peridynamic correspondence model under finite strain assumptions.
```

Comment:
Every place where you reference Form I or Form II, point back to the main page for the description.

`Details of the various forms of the horizon stabilization approaches are provided on the documentation page that describes all [Peridynamics models](PeridynamicModels.md).`